### ISSUE TITLE
Integer overflow in usage request

### ISSUE THREAD

**Issue by lubo-dev** [DeepLcom/deepl-java#44]
If an account sets no character translation limit the limit is 1 000 000 000 000.  
In that case requesting the usage will return a negative number for the character limit.

Minimal example:
```Java
Translator translator = new Translator(authKey)
Usage.Detail characterUsage = translator.getUsage().getCharacter();
characterUsage.getLimit() // returns -727379968
```
The cause of the error lies in `UsageDeserializer::createDetail` line 28 and 29.  
There the `limit` and `count` are parsed as `Integer`, even though they are declared as `long` in `Usage.Detail`.  
Given that the request can return a number larger than `Integer::MAX_VALUE` they should not only be declared as `long` but also parsed as such.

**Comment by lubo-dev** [comment]
I added a pull request for a fix: https://github.com/DeepLcom/deepl-java/pull/45

**Comment by lubo-dev** [comment]
The fix was released with version 1.5.1
